backend/danswer/background/utils.py

```diff
@@ -8,10 +8,13 @@
 logger = setup_logger()
 
 
-def interval_run_job(job: Callable[[], Any], delay: int | float) -> None:
+def interval_run_job(
```

Comment:
Celery Beat angle for sure

Reply:
app.conf.beat_schedule = {
    'my-task': {
        'task': 'myapp.tasks.my_periodic_task',
        'schedule': timedelta(minutes=5),  # Execute every 5 minutes
    },
}

something like this, centralized and CLEAN!